README.md

```diff
@@ -0,0 +1,197 @@
+# pysdccc
+
+This python packages provides a convenient way to execute the [sdccc test suite](https://github.com/Draegerwerk/sdccc/).
+
+This wrapper is only compatible with sdccc versions later than [ebe0e09](https://github.com/Draegerwerk/SDCcc/commit/ebe0e094ff92649d0bda1988b0d1c1b08403aea4).
+
+## Installation
+
+Download from pypi using `pip install pysdccc`
+
+### Development
+
+For this open source project the [Contributor License Agreement](Contributor_License_Agreement.md) governs all relevant activities and your contributions. By contributing to the project you agree to be bound by this Agreement and to licence your work accordingly.
+
+1. clone the repository
+2. install dependencies, e.g. with [`uv sync --dev --all-extras`](https://docs.astral.sh/uv/reference/cli/#uv-sync)
+
+## Usage
+
+### Basic usage
+
+```python
+import subprocess
+import pathlib
+
+import pysdccc
+
+
+def main():
+    if not pysdccc.is_downloaded("my-specific-version"):
+        pysdccc.download("https://url/to/sdccc.zip")
+
+    runner = pysdccc.SdcccRunner(
+        pathlib.Path("/path/to/sdccc/result/directory"),
+    )
+
+    try:
+        return_code, direct_result, invariant_result = runner.run(  # https://github.com/Draegerwerk/SDCcc/?tab=readme-ov-file#exit-codes
+            config=pathlib.Path("/path/to/configuration/file.toml"),
+            requirements=pathlib.Path("/path/to/requirements/file.toml"),
+        )
+    except subprocess.TimeoutExpired:
+        print("Timeout occurred")
+        return
+
+    if direct_result is None or invariant_result is None:
+        print("No result file available")
+        return
+
+    for test_case in direct_result + invariant_result:
+        print(f"{test_case.name}: {test_case.is_passed}")
+```
+If you look for an async version
+
+```python
+import pathlib
+
+import pysdccc
+
+
+async def main():
+    if not await pysdccc.ais_downloaded("my-specific-version"):
+        await pysdccc.adownload("https://url/to/sdccc.zip")
+
+    runner = pysdccc.SdcccRunnerAsync(
+        pathlib.Path("/path/to/sdccc/result/directory"),
+    )
+
+    try:
+        return_code, direct_result, invariant_result = await runner.run(
+            # https://github.com/Draegerwerk/SDCcc/?tab=readme-ov-file#exit-codes
+            config=pathlib.Path("/path/to/configuration/file.toml"),
+            requirements=pathlib.Path("/path/to/requirements/file.toml"),
+        )
+    except TimeoutError:
+        print("Timeout occurred")
+        return
+
+    # checkout example from above ...
+```
+
+### Download an sdccc executable
+
+Check out `pysdccc.download` or `pysdccc.download_async`. Otherwise, if `pysdccc[cli]` is installed, `$ pysdccc install https://url/to/sdccc.zip` can also be used.
```

Comment:
I guess what is meant with pysdccc[cli] but someone else might not. Please explain this, or provide a link.

Maybe like this: "If the command line interface of pysdcc is installed, the zipped source of SDCcc can be installed using this command: `$ pysdccc install https://url/to/sdccc.zip`"

Reply:
done